docs/content/usage/configurations.md

```diff
@@ -97,6 +97,10 @@ Where `database_type` is `sqlite` and `database_database` is the location you wa
 
 ### PostgreSQL Configurations
 
+:::note
+
+Before getting started with configuring PostgreSQL, you should have created a PostgreSQL `database`. You can check how to create a database in [PostgreSQL's documentation](https://www.postgresql.org/docs/current/sql-createdatabase.html).
+
```

Comment:
Please add below this the closing of the admonition:

```
:::
```

Reply:
Fixed